TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -51,11 +119,51 @@ public static class PriorityRoad {
             ushort segmentId = data.CurSeg.segmentId;
             foreach (bool startNode in Constants.ALL_BOOL) {
                 ushort nodeId = netService.GetSegmentNodeId(segmentId, startNode);
-                FixJunction(nodeId);
+                FixHighPrioirtyJunction(nodeId);
             }
             return true;
         }
 
+        /// <summary>
+        /// returns the node of <paramref name="segmentId"/> that is not shared 
+        /// with <paramref name="otherSegmentId"/> .
+        /// </summary>
+        /// <param name="segmentId"></param>
+        /// <param name="otherSegmentId"></param>
```

Comment:
Please don't leave these empty. Even one word is better, like `Segment 1.` Having these empty makes IDE sad.